docs/CHANGELOG.asciidoc

```diff
@@ -44,3 +44,171 @@ This section summarizes the changes in each release.
 
 //[float]
 //=== Known Issues
+
+[[release-notes-7.0.0-alpha1]]
+== {kib} 7.0.0-alpha1
+
+coming[7.0.0-alpha1]
+
+[float]
+[[breaking-7.0.0]]
+=== Breaking changes
+
+For more details about breaking changes in this release, see 
+<<breaking-changes-7.0, Breaking changes in 7.0>>.
+
+Discover::
+* Does not apply `query:queryString:options` to `query_string` filters {pull}15640[#15640]
+* Removes `default_field` from `query:queryString:options` {pull}18966[#18966]
+
+Monitoring::
+* Removes `node_resolver` setting {pull}21181[#21181]
+
+Operations::
+* Removes tribe node support {pull}16397[#16397]
+* Creates separate startup scripts for development and production {pull}13806[#13806]
+* Sets default port based on protocol  {pull}21564[#21564]
+* Removes deprecated `/shorten` API {pull}21861[#21861]
+
+[float]
+[[deprecation-7.0.0]]
+=== Deprecations
+
+Geo::
+* Fixes legacy tilemap loading {pull}22095[#22095]
+
+[float]
+[[K7-design-7.0.0]]
+=== K7 UI Design
+
+For 7.0.0-alpha1, {kib} gives users the ability to switch between the old 6.x 
+style headers and the new K7 design. To turn on the new design, go to 
```

Comment:
The K7 design is the default in these builds, so folks only need to change the advanced setting if they want to switch back to the K6 design. I suggest the following language, but feel free to throw this out and start from scratch if you'd prefer:

> {kib} 7.0.0-alpha1 includes a new design for Kibana called K7. In this early stage, it's still a little rough around the edges, so if for any reason you'd like to switch back to the existing K6 design, head over to *Management > Advanced Settings* and toggle off the `k7design` setting. Note that the option to switch to the old design will be removed before 7.0.0 GA.

Reply:
@epixa I updated the text to reflect your suggestions.